ytools/yast2/create_maintenance_branch

```diff
@@ -29,11 +29,35 @@ SOURCE_COMMIT = ARGV[2] || "origin/master"
 # by default pass output of commands to stdout and stderr
 Cheetah.default_options = { :stdout => STDOUT, :stderr => STDERR }
 
+def project
+  raise unless @project
+
+  @project
+end
+
+NAMESPACES_MAPPING = {
+  "libyui" => "Libyui",
+  "yast"   => "Yast"
+}
+
+def project_namespace
+  raise unless @project
+
+  NAMESPACES_MAPPING[@project] || raise("Unknown project #{@project}")
+end
+
+def project=(value)
+  @project = value
+end
+
 def check_real_upstream
   res = Cheetah.run "git", "remote", "-v", :stdout => :capture
-  if res.lines.grep(/origin\s*git@github.com:yast/).empty?
+  upstream = res.lines.grep(/origin\s*git@github.com:(yast|libyui)/)
+  if upstream.empty?
     raise "This script can work only on upstream clone, where upstream remote is marked as origin"
   end
+
+  self.project = upstream.first[/origin\s*git@github.com:(yast|libyui)/, 1]
```

Comment:
NP: factor out the regexp to a variable, otherwise it is hard to see whether the 2 occurrences are really the same